cpp/ql/src/experimental/Security/CWE/CWE-193/InvalidPointerDeref.ql

```diff
@@ -0,0 +1,316 @@
+/**
+ * @name Invalid pointer dereference
+ * @description Dereferencing a pointer that points past it allocation is undefined behavior
+ *              and may lead to security vulnerabilities.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
```

Comment:
Have you done a MRVA or LGTM run to justify this?

Reply:
I have not. As it's in experimental this doesn't really mean much, though. If you'd prefer I can remove the precision tag. I have not seen any FPs on the query yet, however.

Running this on MRVA is currently blocked because the current image used for MRVA has a compiler bug that the new C++ dataflow library (which the new product dataflow library is using) is hitting.

Reply:
Good point about experimental, and good to hear you haven't seen any FPs.  We should (briefly) review this tag when it comes out of experimental to be sure.